hmac-protected/hmac-protected/handler.py

```diff
@@ -0,0 +1,29 @@
+import os, hmac, hashlib
+
+def validateHMAC(message,secret,hash):
+
+    # The hash starts with the hashing algorithm "sha1=" we need to remove it
```

Comment:
Only true of GitHub, maybe add a helper method for this

Reply:
The `--sign` flag also appends this github style that is why I added the comment